graphql_compiler/compiler/ir_lowering_match/ir_lowering.py

```diff
@@ -93,44 +93,98 @@ def visitor_fn(expression):
     return new_ir_blocks
 
 
-def lower_has_substring_binary_compositions(ir_blocks):
-    """Lower Filter blocks that use the "has_substring" operation into MATCH-representable form."""
+def lower_string_operators(ir_blocks):
+    """Lower Filter blocks that use the "has_substring", "starts_with", or
```

Comment:
This is not a compliant docstring, per https://github.com/kensho-technologies/graphql-compiler/pull/499/files#r316730267

The headline must always fit on one line. The rule is that if you need more than one line to describe the function, then it's not that obvious, and you should stick to a one-line summary + a description underneath + write out `Args` and `Returns` sections as well.